Apps/CocoaPods-FCM/src/View/LoginView.swift

```diff
@@ -0,0 +1,41 @@
+import CioTracking
+import SwiftUI
+
+struct LoginView: View {
+    @State private var firstNameText: String = ""
+    @State private var emailText: String = ""
+
+    @EnvironmentObject var userManager: UserManager
+
+    var body: some View {
+        ZStack {
+            VStack(spacing: 40) { // This view container will be in center of screen.
+                TextField("First name", text: $firstNameText)
+                TextField("Email", text: $emailText)
+                ColorButton(title: "Login") {
+                    CustomerIO.shared.identify(identifier: emailText, body: [
```

Comment:
Maybe it's already in your plan for future PR, but how do you feel about having an injectable `CustomerIOManager` which handles all correspondence and interaction with the CustomerIO library? That would give use of encapsulation.